src/react.js

```diff
@@ -1,15 +1,20 @@
 // @flow
 import * as icons from "./data/icons/react";
-import * as tokenIcons from "./data/icons/react/tokens";
 import type { CryptoCurrency, TokenCurrency } from "./types";
-import iconNameById from "./iconNameById";
 
 type Icon = React$ComponentType<{ size: number, color?: string }>;
 
+function getIconId({ ticker }: CryptoCurrency | TokenCurrency) {
+  let id = ticker.toLowerCase();
+  if (!isNaN(id.charAt(0))) id = `_${id}`; // fix variable name leading with a numerical value
```

Comment:
That approach is dangerous and I fear we will have a lot of mismatch.
Proof: how it's tedious to manage countervalues today, lot of collisions on ticker, etc..

for token only, we may need return `null` when `disableCountervalue` is true.

Reply:
this specific line is to handle tickers that start with a number since creating stuff like that cant work in JS 
```
import { 0xbtc } from "./0xbtc"
```
So i handled it that way since to escape this issue.

The new icons are named via ticker, tokens collision is not really an issue here since the icon color of tokens will be the one from the chain in the live.
Unless we have collision of ticker on currencies itself i dont think its an issue.

As for the scale it has been adapted on the frontend side with these two PR:
https://github.com/LedgerHQ/ledger-live-mobile/pull/1640
https://github.com/LedgerHQ/ledger-live-desktop/pull/3699